### PR TITLE
fixed the wallet assertion

### PR DESCRIPTION
**Pull request by zubair-deriv**
### Card

https://app.clickup.com/t/20696747/WALL-4499

### Issue
1. USD wallet assertion was failing due to a similar crypto eUSD wallet in the list 
2. Updated Derived label to Standard
3. Handle passkey for responsive

### CODE REVIEW

File: cypress/e2e/smoke/Wallets/walletSwitcher.cy.js [hunk 1]
```diff
@@ -52,18 +52,26 @@ function fiatWalletcheck() {
...
+          .invoke('text')
+          .then((text) => {
+            cy.log('text', text)
+            if (text.trim() === 'USD Wallet') {
+              cy.wrap($el)
+                .parent()
+                .parent()
+                .find('button', { timeout: 15000 })
+                .should('contain', 'Add')
+            }
```
Comment (afsaneh-rezaeinejad-deriv) [inline cypress/e2e/smoke/Wallets/walletSwitcher.cy.js:71]: @zubair-deriv we need to verify the USD wallet text is "Added"

**Review by afsaneh-rezaeinejad-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/smoke/Wallets/walletSwitcher.cy.js (hunk 1 above)
Comment (zubair-deriv) [reply cypress/e2e/smoke/Wallets/walletSwitcher.cy.js:73]: @afsaneh-rezaeinejad-deriv Code is updated

**Review by afsaneh-rezaeinejad-deriv — APPROVED** [review]
Looks good

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**